coalib/results/Diff.py

```diff
@@ -70,6 +72,65 @@ def from_string_arrays(cls, file_array_1, file_array_2, rename=False):
         return result
 
     @classmethod
+    def from_unified_diff(cls, unified_diff, original_file):
+        """
+        Creates a `Diff` object from given unified diff.
```

Comment:
use double backticks. Not sure whether RST does support that, but with double backticks we are on the safe side :) (and consistent too)